pgtoolkit/log/parser.py

```diff
@@ -3,10 +3,54 @@
 from datetime import datetime, timedelta
 
 
+class LogParser(object):
```

Comment:
The class only has one method, so it could be a (generator) function. Or do you plan to add more methods later on?